src/firmware/shared/math/matrix.c

```diff
@@ -1,103 +1,97 @@
-#include "matrix.h"
+#include "firmware/shared/math/matrix.h"
 
+#include <assert.h>
 #include <stdlib.h>
```

Comment:
nit: delete since it's included in the header

Reply:
Removed :+1:

Reply:
Added back, turns out I needed it